README.md

```diff
@@ -1 +1,19 @@
-# renode-actions
\ No newline at end of file
+# renode-actions
+
+This repository includes the Renode action for use in Github Actions by testing platforms in the Renode environment
```

Comment:
`This repository contains a Renode GitHub action for testing software in the Renode environment`